src/hotspot/share/opto/superword.cpp

```diff
@@ -4497,14 +4293,18 @@ void SWPointer::Tracer::print_depth() const {
   }
 }
 
+bool SWPointer::Tracer::slp_trace_alignment() {
+  return _slp && _slp->is_trace_alignment();
```

Comment:
Aha, here you wrap it. You have some uses above that could be replaced with this now. But again, even better would be if we had a general trace flag that could trace it for any context, not just SuperWord.

Reply:
After all, should the `VectorizeDebug` flag not apply everywhere? See `phase->C->directive()->VectorizeDebugOption`.

Reply:
I'd also move this to some static functions in a potential "autovectorization.hpp", and move `_vector_loop_debug` there, together with all its `is_trace...` accessors.

Reply:
I agree current code here is a bit ugly. I will try to make it better in `SWPointer` refactoring.

Reply:
Close above comment and leave this open to remind that we should also move `_vector_loop_debug` and all `is_trace...` accessors in a later patch.